Header header
float32[10] offset
float32[10] states
float32[10] pos
int64[10] target1
int64[10] target2
int64[10] target3
int64[10] target4
int64[10] target5
int64[10] target6
int64[10] target7
int64[10] target8
int64[10] target9
int64[10] target10
string IsInPosition
int64 currentLine
